### ISSUE TITLE
Google OpenID Closed

### ISSUE THREAD

**Issue by velkymx** [iignatov/LightOpenID#10]
Is there a plan to add support for the updated Google Auth service?

**Comment by iignatov** [comment]
No, I don't use this library (or PHP at all) anymore, so I have no plans to update it anymore.
